src/lib/pages/contract-details/components/InstantiateInfo.tsx

```diff
@@ -148,7 +148,9 @@ export const InstantiateInfo = ({
 
       <LabelText
         label="Admin Address"
-        helperText1={instantiateInfo.admin ? adminType : undefined}
+        helperText1={
```

Comment:
1. It should look something like `helperText1={getAddressTypeText(adminType)}`.
2. Add the `validator_address` case on the `getAddressTypeText` function, as shown below.
```tsx
export const getAddressTypeText = (addressType: AddressReturnType) => {
  switch (addressType) {
    case "contract_address":
      return "(Contract Address)";
    case "user_address":
      return "(Wallet Address)";
    case "validator_address":
      return "(Validator Address)";
    case "invalid_address":
    default:
      return "(Invalid Address)";
  }
};
```